### PR TITLE
esn-frontend-inbox#53: Fixed Linshare Uploader Icons

### LINKED ISSUE #53: Bring back the Linshare upload provider when composing an email
att2
att1

TODOs:

- [x] 1. Bring back `linagora.esn.linshare` in a repository named `esn-frontend-linshare`.
- [x] 2. Bring back `linagora.esn.unifiedinbox.linshare` in a repository named `esn-frontend-linshare-inbox`. The Inbox SPA needs to require this.
- [x] 3. Move all shared dependencies in `linagora.esn.unifiedinbox` to `esn.inbox.libs`.
- [ ] 4. Remove all dynamic directives and injecting the LinShare attachment button into the mail composer in `esn-frontend-inbox`.
- [x] 5. Check if everything works by:
     - [x] 5.1. Setting up LinShare's backend
     - [x] 5.2. Registering OpenPaaS' Public Key into LinShare
- [ ] 6. Add translation to `esn-frontend-linshare` and `esn-frontend-inbox-linshare`.

### PR DESCRIPTION
**Pull request by itswillta**
Partially resolves https://github.com/OpenPaaS-Suite/esn-frontend-inbox/issues/53. Please refer to https://github.com/OpenPaaS-Suite/esn-frontend-inbox/pull/59 for the demo.

### CODE REVIEW

**Review by tuanlc — APPROVED** [review]

File: src/images/linshare-desktop-icon.svg [hunk 1]
```diff
@@ -1,79 +1,75 @@
...
-<!-- Generator: Adobe Illustrator 15.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
-<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
-<svg version="1.1" id="Calque_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
-	 width="15.357px" height="23.893px" viewBox="0 0 15.357 23.893" enable-background="new 0 0 15.357 23.893" xml:space="preserve">
-<g id="Calque_1_1_" display="none">
-	<g display="inline">
-		<rect id="Calendar_3_" x="-26.365" y="0.045" fill="none" width="23.897" height="23.897"/>
-		<g>
-			<path id="calendar_3_" fill="#2196F3" d="M-9.534,17.623h-9.839V9.892h9.839 M-11.64,4.971v1.406h-5.623V4.971h-1.406v1.406
+<svg version="1.1" id="Calque_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="15.357px" height="23.893px" viewBox="0 0 15.357 23.893" enable-background="new 0 0 15.357 23.893" xml:space="preserve">
```
Comment (piavgh) [inline src/images/linshare-desktop-icon.svg:1]: Why did you change these svg files?

Comment (itswillta) [reply src/images/linshare-desktop-icon.svg:1]: I just reformatted (prettified) them and removed some unnecessary metadata, as they are directly injected in HTML as inline SVGs now.

**Review by piavgh — APPROVED** [review]
